spec/services/sign_in/token_serializer_spec.rb

```diff
@@ -116,16 +116,19 @@
         end
       end
 
-      context 'and client is not configured to check for anti csrf' do
-        let(:anti_csrf) { false }
+      context 'when client_config has refresh_token_path' do
+        let(:client_config) { create(:client_config, authentication:, refresh_token_path: refresh_path) }
 
-        it 'does not anti csrf token cookie' do
-          subject
-          expect(cookies[anti_csrf_token_cookie_name]).to eq(nil)
+        it 'returns the refresh_token_path from client_config' do
+          expect(client_config.refresh_token_path).to eq(refresh_path)
         end
+      end
 
-        it 'returns an empty hash' do
-          expect(subject).to eq({})
+      context 'when client_config does not have refresh_token_path' do
+        let(:client_config) { create(:client_config, authentication:, refresh_token_path: '') }
+
+        it 'returns nil when refresh_token is not present' do
+          expect(client_config.refresh_token_path).to eq('')
```

Comment:
same comments as above. you should be testing the `token_serializer` when `client_config.refresh_path` is `nil`. also, `''` is a string not `nil`